### DISCUSSION TITLE
Strange problem: once working now not - OSX 11.5.2

### DISCUSSION THREAD

**Discussion by grimmwerks** [chris1111/Wireless-USB-Big-Sur-Adapter#142] [Q&A]
Ok so my son - 9 - had my old macbook which has a wifi antenna problem, so i've been buying USB adapters for him to use.  Found this project and it was a godsend at first, but for some reason now is not working.    

First the Cudy WU1300S (https://www.amazon.com/gp/product/B084FS7BWF/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1) was working fine until it didn't.  Ok, so thought it was the adapter.

Bought the BT+AC600 and that was working fabulously for awhile - but his younger brother also uses the macbook (they're both making roblox games already -- hooray Lua!) - and I noticed that if you logged into another user the first user wouldn't give up the wifi adapter.

After awhile this too stopped working, so I bought a second one as the adapter didn't even light up anymore - thought it failed.  The new one didn't light up / work / unseen on the usb sys info.  The USB is def working as I could mount disks and also had the iPhone as wifi in a test.

I finally bought this one https://www.amazon.com/gp/product/B08QCDR6V6/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1 and it too is unrecognized.

What am I doing wrong, or what could've happened that this is no longer working?  I've used it since  .7,  upgraded to .9 and it worked for a bit, but haven't gotten 10 to work at all.

Edit: 
These are definitely seen by the OS:

<img width="460" alt="Screen Shot 2021-09-15 at 1 34 31 PM" src="https://user-images.githubusercontent.com/210795/133482378-66cb49e0-5c45-4936-ac4b-5dba3af07edf.png">
<img width="511" alt="Screen Shot 2021-09-15 at 1 33 59 PM" src="https://user-images.githubusercontent.com/210795/133482387-3790ae0f-e272-41e3-9806-29ae1c64d8c3.png">

**Comment by grimmwerks** [comment]
I am an idiot; this absolutely worked for me: https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/issues/131

Edit: well... worked on first reboot, second reboot it can't see the device again.  Don't have an option to allow realtek again.  Investigating.

**Comment by grimmwerks** [comment]
Strangely enough after it worked once and didn't work again, now the adapter isn't being seen via usb.  Very strange.

**Comment by chris1111** [comment]
By a better compatible adapter, see working list

**Reply by grimmwerks** [reply]
The Cudy WU1300S was the first one I got - on the list, works sometimes, then not seen by USB.

**Comment by chris1111** [comment]
CUDY WU1300S Officiallly report by User as working adpater
https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/commit/162f87ae6788a8facf492a605f887c21eee46445
